pkg/operator/ceph/config/monstore.go

```diff
@@ -236,7 +236,7 @@ func (m *MonStore) SetAll(clientName string, settings map[string]string) error {
 
 	out, err := cephCmd.RunWithTimeout(exec.CephCommandsTimeout)
 	if err != nil {
-		logger.Errorf("failed to run command ceph %s", args)
+		logger.Errorf("failed to run command ceph %s: %v", args, err)
```

Comment:
Definitely a useful logging update, just a small formatting nit. And since this isn't directly related to the compression issue, could you add this to a separate commit? 
```suggestion
		logger.Errorf("failed to run command ceph %s. %v", args, err)
```

Reply:
I've added a separate commit for this change as requested @travisn. I've left the colon, if that's okay.

I believe this is the convention when following effective Go and the Go code review comments.

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
The colon is fine since it's just logging immediately. Where rook appends errors to existing errors, we just have a pattern of separating with a `.`.